app/src/main/java/com/duckduckgo/app/browser/omnibar/Omnibar.kt

```diff
@@ -358,4 +384,24 @@ class Omnibar(
     fun voiceSearchDisabled(url: String?) {
         newOmnibar.decorate(DisableVoiceSearch(url ?: ""))
     }
+
+    override fun onScrollChange(
```

Comment:
the new omnibar doesn’t hide, it shrinks as the user scrolls and a mini-bar appears. we use the scroll of the webView to control that. this function will pass the scroll to the omnibar if the new experimental ui is enabled.